src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3254,6 +3255,32 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         return _terminal->GetTaskbarProgress();
     }
 
+    winrt::fire_and_forget TermControl::InvertScreenColors()
+    {
+        if (!_invertTimer)
+        {
+            co_await winrt::resume_foreground(Dispatcher());
+
+            DispatcherTimer invertTimer;
+            invertTimer.Interval(std::chrono::milliseconds(2000));
+            invertTimer.Tick({ get_weak(), &TermControl::_InvertTimerTick });
+            invertTimer.Start();
+            _invertTimer.emplace(std::move(invertTimer));
+            _terminal->SetScreenMode(true);

```

Comment:
Do we possibly need to stash the original `_screenReversed` value of the `Terminal`, invert the current value, and only restore to the original value if the value hasn't changed during the blink? I'm imagining a weird race like:

* the client app reverses the screen colors
* emits a BEL
* then reverses the colors back to the original, before 2000ms have passed

or even if the app was already in reversed screen mode, then emitted a BEL. They'd probably want to blink to the _normal_ presentation, then come back to the reversed appearance. (That's weird, sure, but I'm sure there's some weird app out there that does this).

Reply:
Good call, we stash the value now